README.md

```diff
@@ -80,6 +80,9 @@ Download `AmazonChimeSDKMedia` binary with bitcode support from the latest [rele
 
 Unzip and copy the `AmazonChimeSDKMedia.framework` to `AmazonChimeSDK` folder.
 
+Note: For Xcode12.3 and later, Amazon Chime SDK iOS v0.15.0 and later, you can place unzipped AmazonChimeSDKMedia.xcframework to `AmazonChimeSDK` folder. For each target, remove AmazonChimeSDKMedia.framework and add AmazonChimeSDKMedia.xcframework.
```

Comment:
We should be more explicit about using XCFramework if traditional framework causes compile error.
Instead of "you can place..." -> "please replace .framework with .xcframework by: "